.github/config/evergreen-arm-hardfp-rdk.json

```diff
@@ -0,0 +1,40 @@
+{
+  "docker_service": "build-raspi",
```

Comment:
I just realized that we should put a note in the commit message about using the raspi builder too. Can you also create a tracking bug to follow up on this later?

Reply:
Updated the commit message and b/307558214 tracks this rdk-only build platform task.